wb-codemod/package.json

```diff
@@ -1,10 +1,18 @@
 {
   "name": "@khanacademy/wb-codemod",
-  "version": "2.1.1",
   "license": "MIT",
+  "version": "2.1.1",
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
The `.changeset/config.json` actually overrides this, but I think it's useful to have this defined here instead of relying on `changesets` magic override.